docs/BOOTING.md

```diff
@@ -79,9 +79,45 @@ Second stage GRUB is expected to do all the [heavy lifting](../pkg/grub/rootfs.c
 an EVE instance and because it resides in IMGA or IMGB partitions it can easily be upgraded and patched.
 Behavior of this stage of boot process is controlled by a read-only grub.cfg under {IMGA,IMGB}/EFI/BOOT/
 but it can further be tweaked by the grub.cfg overrides on the CONFIG partition. Note that an override
-grub.cfg is expected to be a [complete override](../pkg/grub/rootfs.cfg#L132) and anyone constructing
+grub.cfg is expected to be a [complete override](../pkg/grub/rootfs.cfg#L143) and anyone constructing
 its content is expected to be familiar with the overall flow of read-only grub.cfg.
 
+Option to use in grub.cfg on the CONFIG partition:
+
+1. Cgroup related options. Please follow the [example](README.md#eve-cgroups) to write
+   these options in the correct variables:
+    1. `dom0_mem` - memory limit for eve cgroup (default is `750M,max:750M`)
+    2. `dom0_max_vcpus` - cpu limit for eve cgroup (default is `1`)
+    3. `eve_mem` - memory limit for cgroups with services of EVE (default is `600M,max:600M`)
+    4. `eve_max_vcpus` - cpu limit for cgroups with services of EVE (default is `1`)
+    5. `ctrd_mem` - memory limit for cgroups with containerd-shims of EVE (default is `350M,max:350M`)
+    6. `ctrd_max_vcpus` - cpu limit for cgroups with containerd-shims of EVE (default is `1`)
+2. Installer options. These options are used during [the installation](DEPLOYMENT.md) process and may be set by adding
+   line `set_global dom0_extra_args "$dom0_extra_args OPTION1=VAL1 OPTION2 "`, where OPTION fields described below
```

Comment:
I think this business about how to pass options to GRUB should be at the top of this new blurb. This will make even more sense if we fix the above section to properly refer to things like hv_dom0_mem_settings, etc.

Reply:
I put some information on the top